model/vowpalWabbit/examples/accuracy.sh

```diff
@@ -0,0 +1,9 @@
+dffml accuracy  \
+  -model vwmodel \
```

Comment:
Following the above example

```suggestion
  -model vwmodel \
  -model-noconvert \
```